tests/tests.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+import math
+
+from .homework import Rectangle
+
+
+class RectangleTestCase(unittest.TestCase):
+    test_rectangle_1 = Rectangle(1, 4)
+    test_rectangle_2 = Rectangle(10, 3)
+    test_rectangle_3 = Rectangle(5, 5)
+
+    def test_rectangle_perimeter_calc(self):
+        self.assertEqual(self.test_rectangle_1.get_rectangle_perimeter(), 10)
+        self.assertEqual(self.test_rectangle_2.get_rectangle_perimeter(), 26)
+        self.assertEqual(self.test_rectangle_3.get_rectangle_perimeter(), 20)
+
+    def test_rectangle_square_calc(self):
+        self.assertEqual(self.test_rectangle_1.get_rectangle_square(), 4)
+        self.assertEqual(self.test_rectangle_2.get_rectangle_square(), 30)
+        self.assertEqual(self.test_rectangle_3.get_rectangle_square(), 25)
+
+    def test_sum_of_corners_calc(self):
+        for i in range(1, 5):
+            with self.subTest(name=f'sum of {i} corners'):
```

Comment:
do you still need subtests here?

Reply:
idk it looks cool in pycharm :)
![image](https://user-images.githubusercontent.com/46224733/66190109-f5354200-e693-11e9-8a19-1cfd5261a670.png)

Reply:
but okay I will remove subtests. thanks